src/components/App/App.jsx

```diff
@@ -30,21 +32,36 @@ const App = () => {
   })
 
   if (loading) {
-    return <Loading />;
+    return <Loading />
   }
 
   return (
     <div id="app" className="d-flex flex-column h-100">
```

Comment:
Branch Feature/DocsPage has made some changes, it may conflict here.

Reply:
You and Alexis can probably do a final pull down from master and staging before you do your PR to get everything in sync.